### ISSUE TITLE
Movies Anywhere

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#456]
_Instructions: This template is written in Markdown. Click the "Preview" tab to see what it will look like when submitted. Fill in the applicable sections (you can erase the examples if you'd like). For more information, please see [this page](https://github.com/richardfrost/AdvancedProfanityFilter/wiki/New-Audio-Muting-Site)._

#### :movie_camera: The Site
[moviesanywhere.com](https://moviesanywhere.com)
#### :speech_balloon: Subtitle Element/Node
This site uses HTML5 Video TextTracks

#### :pencil: Additional notes
Requested by @WolfD1g1tal

**Comment by richardfrost** [comment]
After testing on several titles this config seems to work well:

```json
{
  "moviesanywhere.com": [
    {
      "mode": "cue"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

For best results, make sure to remove any custom config used for testing like this once its been included in the official update.

I'll get this added in the next release.

**Comment by richardfrost** [comment]
This has been included in [version 2.39.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.39.0). Feel free to comment here if you have any feedback for this update.
